pages/doc/wavefront_release_notes.md

```diff
@@ -26,21 +26,55 @@ For information about the two subscription types and how they differ, see [Subsc
  
 {% include note.html content="We will support both original and onboarded subscriptions until all original subscriptions are migrated to VMware Cloud services."%}
 
-
-## 2023-29.x Release Notes
+## 2023-31.x Release Notes
 
 ### Original and Onboarded Subscriptions
 
-<!-- Uncomment and release for 20223-30.x - next release
+* **Logs GA release**: This release marks the GA release for the [Logs feature](logging_overview.html). You can:
+
+{% include important.html content="Logs is enabled only for selected customers. If you like to participate, contact your account representative or [technical support](wavefront_support_feedback.html#support)." %}
+  * [Send your logs](logging_send_logs.html) to our service, and troubleshoot using the [Logs Browser](logging_log_browser.html).
+  * [Create logs charts](logging_logs_chart.html) to view logs next to your metrics data.
+  * Drill into logs from alerts, charts, and traces.
+  * [Customize the default log settings](logging_logs_settings.html) to map logs tags to metrics and trace tags so you can drill into logs seamlessly.
+  
+  ![A screenshot of the logs browser.](images/logs_browser_ga_for_release_notes.png)
+
+* **Logs Alerts**: You can create alerts for your logs data and see the firing events of the logs alert. See [Manage Logs Alerts](logging_log_alerts.html) for details.
+  {{site.data.alerts.note}}
+  <ul>
+      <li>
+          You need the <b>Logs</b> permission to view the logs alerts.
+      </li>
+      <li>
+          You need the <b>Alerts</b> and <b>Logs</b> permissions to create and manage logs alerts.
+      </li>
+      <p>The steps to add roles and permissions differ for Onboarded and Original subscriptions. See add permissions details for <a href="csp_users_roles.html#create-edit-or-delete-a-custom-role">Onboarded subscriptions</a> and <a href="users_roles.html">Original subscriptions</a>.</p>
```

Comment:
The steps to add roles and permissions differ for onboarded and original subscriptions. For details, see how to manage roles, groups, and permissions for <a href="csp_users_roles.html#create-edit-or-delete-a-custom-role">onboarded subscriptions</a> and <a href="users_roles.html">original subscriptions</a>.